lib/domain/service/call.dart

```diff
@@ -168,6 +168,9 @@ class CallService extends DisposableService {
           withVideo: withVideo,
           withScreen: withScreen,
         );
+        if (chat?.unreadCount.value == 1) {
+          _chatService.readAll([chatId]).ignore();
```

Comment:
`ignore()` is bad practice. It ignores all the errors happening in that `Future`. Can't we at least do a `Log.error` when this method fails so that we can see in the logs the error, if any? Take a look at the documentation: https://api.flutter.dev/flutter/dart-async/FutureExtensions/ignore.html